osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/deploymentspec/OnboardDAITask.java

```diff
@@ -77,16 +80,19 @@ public void executeTransaction(EntityManager em) throws Exception {
                 egressPort.setParentId(domainId);
                 if (domainId != null){
                 	//Element Object is not used in DefaultInstepctionPort for now, hence null
-                    controller.registerInspectionPort(new DefaultInspectionPort(ingressPort, egressPort, null));
+                    VirtualSystem vs = ds.getVirtualSystem();
+                    DistributedAppliance da = vs.getDistributedAppliance();
+                    String redirectionTargetId = da.getRedirectionTargetId();
```

Comment:
you have to do ds.getRedirectionTarget here.

also we should unify this now. We have to call register inspection port always as long as there is controller.